grafana/risingwave-dashboard.dashboard.py

```diff
@@ -2183,6 +2182,49 @@ def section_grpc_hummock_meta_client(outer_panels):
         ),
     ]
 
+def section_memory_manager(outer_panels):
+    panels = outer_panels.sub_panel()
+    return [
+        outer_panels.row_collapsed(
+            "Memory manager",
+            [
+                panels.timeseries_count(
+                    "LRU manager loop count",
+                    "",
+                    [
+                        panels.target(
+                            f"{metric('lru_runtime_loop_count')}",
```

Comment:
`rate` might be a better choice?

```suggestion
                            f"rate({metric('lru_runtime_loop_count')}[$__rate_interval])",
```

Reply:
Okay